src/schedule/MongoSchedule.ts

```diff
@@ -27,7 +27,21 @@ export class MongoSchedule extends Schedule {
     protected readonly scheduleId: string,
     protected readonly connection: Connection,
     pingIntervalMs: number,
+    maxPingRetries: number,
+    retryIntervalMs: number,
   ) {
+    if (!Number.isFinite(pingIntervalMs) || pingIntervalMs < 1) {
```

Comment:
Hmm... this class is never really meant to be instantiated via the the constructor, rather exclusively via the `MongoScheduleBuilder` or the `MongoSchedule.connect` method, right? I've never been a fan of throwing runtime errors in constructors, but here it feels like it might actually fit. What do you think @luipe , @weissu42 ?